server/httpin/restapi/app.go

```diff
@@ -59,14 +59,19 @@ func (a *restapi) handleInstallApp(w http.ResponseWriter, r *http.Request, in pr
 		httputils.WriteError(w, errors.Wrap(err, "failed to unmarshal manifest"))
 		return
 	}
+	deployAs := m.MustDeployAs()
+	if deployAs == "" {
+		httputils.WriteError(w, utils.NewInvalidError("install API does not yet support multiple deploy types"))
```

Comment:
Follow up ticket?

Reply:
Let's wait, I am still unclear on what the actual use-case and the flow for the plugin- (or otherwise) initiated `InstallApp`. Adding a `deployType` parameter and using it here would not be particularly difficult but may necessitate restructuring the API; don't want to do it prematurely.